nautobot/core/settings.py

```diff
@@ -445,6 +446,7 @@
     "nautobot.core.middleware.ExternalAuthMiddleware",
     "nautobot.core.middleware.ObjectChangeMiddleware",
     "django_prometheus.middleware.PrometheusAfterMiddleware",
+    "silk.middleware.SilkyMiddleware",
```

Comment:
Just want to make sure you checked that the order of the middleware is ok. From https://github.com/jazzband/django-silk?tab=readme-ov-file#installation

> Note: The middleware placement is sensitive. If the middleware before silk.middleware.SilkyMiddleware returns from process_request then SilkyMiddleware will never get the chance to execute. Therefore you must ensure that any middleware placed before never returns anything from process_request. See the [django docs](https://docs.djangoproject.com/en/dev/topics/http/middleware/#process-request) for more information on this.

Reply:
Good call out. Technically none of our current middleware are problematic, but no reason to not move it towards the top. Side note, we should investigate if the GZip middleware would be useful to us.